src/file/bmp_image.cpp

```diff
@@ -575,14 +575,15 @@ namespace Bitmap_Operation
 
         const uint8_t * imageY = image.data() + (startY + height - 1) * rowSize + startX * colorCount;
 
-        std::vector < uint8_t > temp( lineLength, 0 );
-
         const size_t imageLineSize = sizeof( uint8_t ) * width * colorCount;
 
+        uint8_t alignmentDiff = static_cast<uint8_t>(lineLength - imageLineSize);
```

Comment:
Please make this variable const and also it should be std::streamSize type to avoid extra type conversation.

Reply:
I made the requested changes